client/src/plugins/camunda-plugin/deployment-tool/DeploymentTool.js

```diff
@@ -135,7 +135,11 @@ export default class DeploymentTool extends PureComponent {
     try {
 
       // (3.1) Retrieve version we deploy to via API
-      version = (await this.getVersion(configuration)).version;
+      try {
+        version = (await this.getVersion(configuration)).version;
+      } catch (error) {
+        version = null;
+      }
```

Comment:
This makes the function swallow all errors. I'd rather like to avoid a situation when there's a bug and we don't know about it because no errors are thrown. Can you please make sure that we ignore the version only in case of 404 errors?

Reply:
Also: We're missing test coverage here.